### PR TITLE
chore: clean up issue templates

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

Clean up and unify formatting on our issue templates. This is in preparation for better automated issue triage work via automation in the future. 

cc @MarshallOfSound 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ about: Create a report to help us improve Electron
...
 ---
 
-* **Electron Version** (output of `node_modules/.bin/electron --version`):
-  * <!-- e.g. 4.0.3 -->
-* **Operating System** (Platform and Version):
-  * <!-- e.g. macOS 10.13.6 / Windows 10 (1803) / Ubuntu 18.04 x64 -->
-* **Last known working Electron version** (if applicable):
-  * <!-- e.g. 3.1.0 -->
+* **Electron Version:**
+<!-- (output of `node_modules/.bin/electron --version`) e.g. 4.0.3 -->
```
Comment (nornagon) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: The initial `*` here was to make it visually stand out to a human. Can we keep it? I like moving the `--version` hint inside the comment though.

File: .github/ISSUE_TEMPLATE/security_report.md [hunk 2]
```diff
@@ -4,6 +4,4 @@ about: Do not create an issue for security reports, send an email to security@el
 
 ---
 
-Do not create an issue for security reports, send an email to:
-
-security@electronjs.org
+**DO NOT** create an issue for security reports, send an email to: [security@electronjs.org](mailto:security@electronjs.org)
```
Comment (nornagon) [inline .github/ISSUE_TEMPLATE/security_report.md]: I don't think the markdown formatting improves things here, as the issue template will be shown in plain text.

Comment (codebytere) [reply .github/ISSUE_TEMPLATE/security_report.md]: i'll remove the mailto!

File: .github/ISSUE_TEMPLATE/Bug_report.md (hunk 1 above)
Comment (codebytere) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: done!

**Pull request merged**